test/CMakeLists.txt

```diff
@@ -367,6 +367,11 @@ if (TARGET TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_allocators DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_arena_priorities DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_dynamic_link DEPENDENCIES TBB::tbb)
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
Should we check on LTO also?

Reply:
It doesn't break the non-LTO build.

For correct check it is necessary to check TBB flags: ``TBB_IPO_PROPERTY, TBB_IPO_FLAGS`` and Cmake flags: ``CMAKE_INTERPROCEDURAL_OPTIMIZATION, DCMAKE_INTERPROCEDURAL_OPTIMIZATION_<config>``.
It's necessary?